<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Project</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>folder-yellow.png</normaloff>folder-yellow.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="LineEditPath"/>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonDir">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>folder-yellow.png</normaloff>folder-yellow.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Project Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LineEditProjekt"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="ButtonCreate">
     <property name="minimumSize">
      <size>
       <width>141</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
